### PR TITLE
Fixed run access page

### PR DESCRIPTION
**Pull request by Grisou13**
w

### CODE REVIEW

File: runners.config.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "api": "http://SC-C333-PC08/api"
+  "api": "https://markal.servehttp.com/runners/api"
```
Comment (kayoumido) [inline runners.config.json]: Which one do we use?

File: src/models/vehicle.ts [hunk 2]
```diff
@@ -81,12 +81,13 @@ export class Vehicle {
...
     v.id = data.id || data._id;
-    v.num = data.num;
+    // v.num = data.num;
+    v.name = data.name;
     v.type = VehicleCategory.build(data.type);
     v.plateNumber = data.plate_number;
     v.nbPlace = data.nb_place;
     v.user = User.build(data.user);
-
+    console.log(v)
```
Comment (kayoumido) [inline src/models/vehicle.ts]: Debug code? If yes remove

Comment (Grisou13) [reply src/models/vehicle.ts]: Just for the fun

File: runners.config.json (hunk 1 above)
Comment (Grisou13) [reply runners.config.json]: http://sit2a.cf/api

Comment (phurni) [reply runners.config.json]: Reference an ENV variable here, so that changing the target server doesn't impact the repo!

**Pull request merged**